conference/cfp.py

```diff
@@ -164,6 +164,44 @@ def update_proposal(request, talk_uuid):
     })
 
 
+@login_required
+def update_speakers(request, talk_uuid):
+    """
+    Update/Edit proposal's speaker(s) view
+    """
+    talk = get_object_or_404(Talk, uuid=talk_uuid)
+
+    if not talk.created_by == request.user:
+        return HttpResponseForbidden()
+
+    conf = Conference.objects.current()
+    if not conf.cfp():
+        return HttpResponseForbidden()
+
+    speaker_form = AddSpeakerToTalkForm(
+        initial=extract_initial_speaker_data_from_user(request.user)
+    )
+
+    if request.method == 'POST':
+        speaker_form = AddSpeakerToTalkForm(request.POST)
```

Comment:
I think in this case we might want to update the form name (or created an alias for this form with some other name that better suits this view)